ui/components/Policies/Utilis/Parameters.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { PolicyValidationParam } from "../../../lib/api/core/core.pb";
+import { ParameterCell, ParameterWrapper, parseValue } from "./PolicyUtilis";
+
+const Parameters = ({
```

Comment:
Withiin this PR, there are a couple different ways that components are being declared. Some via `const` and some via `function`.

Any thoughts on the tradeoffs and is it worth making them consistent?

Reply:
`Severity` is the only function and it's exported as default as there's no other fn in the file to export.